core/server/api_container/server/startosis_engine/interpretation_time_value_store/service_value_repository_test.go

```diff
@@ -62,6 +63,44 @@ func TestPutGetFail_ForMissingServiceName(t *testing.T) {
 	require.Nil(t, actualService)
 }
 
+func TestGetServices(t *testing.T) {
+	repository := getServiceInterpretationTimeValueRepository(t)
+	require.NotNil(t, repository)
+
+	applicationProtocol := ""
+	maybeUrl := ""
+
+	port, interpretationErr := port_spec.CreatePortSpecUsingGoValues(
+		string(serviceName),
+		uint16(443),
+		port_spec_core.TransportProtocol_TCP,
+		&applicationProtocol,
+		"10s",
+		&maybeUrl,
+	)
+	require.Nil(t, interpretationErr)
+	ports := starlark.NewDict(1)
+	require.NoError(t, ports.SetKey(starlark.String("http"), port))
+
+	serviceOneName := fmt.Sprintf("%v-%v", serviceNameStarlarkStr, "1")
+	serviceOne, interpretationErr := kurtosis_types.CreateService(starlark.String(serviceOneName), hostName, ipAddress, ports)
+	require.Nil(t, interpretationErr)
+
+	serviceTwoName := fmt.Sprintf("%v-%v", serviceNameStarlarkStr, "2")
+	serviceTwo, interpretationErr := kurtosis_types.CreateService(starlark.String(serviceTwoName), hostName, ipAddress, ports)
+	require.Nil(t, interpretationErr)
+
+	err := repository.PutService(service.ServiceName(serviceOneName), serviceOne)
+	require.Nil(t, err)
+	err = repository.PutService(service.ServiceName(serviceTwoName), serviceTwo)
+	require.Nil(t, err)
+
+	actualServices, err := repository.GetServices()
+	require.NoError(t, err)
+	require.Len(t, actualServices, 2)
+	// TODO: check the contents of each objects
+}
```

Comment:
just assert that serviceOne is the same as actualServices[0]?

Reply:
oops this wasn't meant to be left as a `TODO` def need to address